style.css

```diff
@@ -369,17 +407,157 @@ hr {
   padding: 12px 16px;
 }
 
+.nav-btn.active li a {
+  font-size: 32px;
+  color: white;
+  font-weight: 600;
+  line-height: 44px;
+  margin-bottom: 20px;
+}
+
+#preview {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  background-color: rgba(193, 199, 208, 0.9);
+  backdrop-filter: blur(5px);
+  z-index: 7;
+  padding: 10px;
+  margin: 0 auto;
+}
+
+.preview-project {
+  position: fixed;
+  top: 47%;
+  left: 47%;
+  transform: translate(-50%, -50%);
+  box-sizing: border-box;
+  background-color: white;
+  width: 70%;
+  border-radius: 10px;
+  margin: 10px auto;
+  padding: 1rem;
+}
+
+.pre-project-info {
+  display: grid;
+  justify-content: space-between;
+  grid-template-columns: 60% 40%;
+  margin: 0 auto;
+}
+
+.pre-project-info hr {
+  border-top: 2px solid #ebecf0;
+  margin: 5px 0 20px 0;
+}
+
+.preview-project-title {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 5px 0;
+}
+
+.preview-title-div {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.preview-project-xmark {
+  position: relative;
+  bottom: 15px;
+  font-size: 14px;
+  border: none;
+  background-color: white;
+  color: #67798e;
+  cursor: pointer;
+}
+
+.preview-project-img {
+  width: 100%;
+  height: 53vh;
+}
```

Comment:
I try my best to reduce the size of the image I had tow options for making that image no to lose it's quality 
- reducing the popup window width size
- no to set both value width or height value for the image 
for the first option if I reduce the pop up window width size since the pop up window position is fixed the top and bottom part of the pop up window was going over the screen and if I did not set both value height and width it was not taking the exact same space as it is in the Figma. in the Figma design the image height for the pop up window is less than the image for the projects images and the screen size totally different when I try with that specific image it was fitting nice but since the height of the image for the projects is more it not looking the same.

Reply:
dear sir the projects images height is totally different with the height of the Figma pop up window image height size and also you can see the size of the window width size in Figma is different with window width size of the desktop if I want to improve the image quality I must not set both height value for the pop up window image when I not set the height of the pop up window the height size is more the pop up window top and bottom part is going of the screen because of the image hight another way is to decrease the pop up window width in that case i should set the value of the width up to 30% or 35% which will look completely different with Figma design.